src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.html

```diff
@@ -0,0 +1,171 @@
+          successText="Snapshot created"
+          errorText="Snapshot error"
+          defaultText="Snapshot"
+        ></app-exporter-step>
+        <app-exporter-step [stepState]="state.doi" aiIcon="ai-doi" successText="DOI created" errorText="DOI error" defaultText="DOI">
+        </app-exporter-step>
+        <app-exporter-step
+          *ngIf="action === SnapshotExporterAction.ORCID"
+          [stepState]="state.orcid"
+          [faIcon]="faOrcid"
+          successText="ORCID created"
+          errorText="ORCID error"
+          defaultText="ORCID"
+        ></app-exporter-step>
+      </div>
+
+      <ng-template *ngTemplateOutlet="zenodoCheck"></ng-template>
+      <div *ngIf="action === SnapshotExporterAction.ORCID">
+        <ng-template *ngTemplateOutlet="orcidCheck"></ng-template>
+      </div>
+
+      <div *ngIf="action === SnapshotExporterAction.DOI">
+        <ng-template *ngTemplateOutlet="doiBlurb"></ng-template>
+      </div>
+      <div *ngIf="action === SnapshotExporterAction.ORCID">
+        <ng-template *ngTemplateOutlet="orcidBlurb"></ng-template>
+      </div>
+
+      <div mat-dialog-actions fxLayout="row" fxLayoutAlign="end">
+        <div
+          *ngIf="promptToConfirmSnapshot && state.overall === StepState.INITIAL"
+          fxLayout="row"
+          fxLayoutAlign="end"
+          fxFlex="50"
+          fxFlexOffset="50"
+        >
+          <span class="alert-warning">Are you sure you want to snapshot this version, making it uneditable? This cannot be undone.</span>
+        </div>
+        <div fxLayoutAlign="end" fxLayout="row" fxFlex="50">
+          <button mat-button mat-dialog-close color="secondary">Cancel</button>
+          <button
+            [disabled]="
+              (hasZenodoToken$ | async) !== true ||
+              (action === SnapshotExporterAction.ORCID && (hasOrcidToken$ | async) !== true) ||
+              state.overall !== StepState.INITIAL
+            "
+            color="primary"
+            mat-flat-button
+            data-cy="export-button"
+            (click)="export(false)"
+            *ngIf="!promptToConfirmSnapshot"
+          >
+            {{ action === SnapshotExporterAction.DOI ? 'Request DOI' : 'Export' }}
+          </button>
+          <button
+            color="primary"
+            mat-flat-button
+            [disabled]="state.overall !== StepState.INITIAL"
+            (click)="export(true)"
+            *ngIf="promptToConfirmSnapshot"
+          >
+            Confirm
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <ng-template #snapshotBlurb>
+    <p>
+      A snapshot freezes the workflow version, ensuring that the content of the workflow does not change, e.g., if cited in publications.
+      <strong>Important note: once a snapshot is made, the workflow version becomes uneditable, apart from some metadata.</strong>
+    </p>
+    <p>
+      You can then use this snapshot to request a DOI or export to your ORCID profile.
```

Comment:
You're not supposed to use the word profile. See the `When talking or writing about ORCID` section https://info.orcid.org/brand-guidelines/

Reply:
Huh, thanks for pointing this out. Will keep a mental note.